vllm/v1/attention/backends/triton_attn.py

```diff
@@ -337,7 +341,7 @@ def forward(
                     layer._v_scale,
                 )
             else:
-                torch.ops._C_cache_ops.reshape_and_cache_flash(
+                ops.reshape_and_cache_flash(
```

Comment:
Is it correct that this is the only part of the Triton backend that needs customization for XPU? It doesn't need to block this PR but we also have some high performant Triton implementations for reshape_and_cache. It would probably make sense to integrate those so that the Triton backend doesn't have this platform-specific dependency.

Reply:
thanks for reminder, can you point where is the _Triton implementations for reshape_and_cache_